hack/push-container-manifest.sh

```diff
@@ -34,7 +34,12 @@ function podman_push_manifest() {
     ${KUBEVIRT_CRI} manifest create ${DOCKER_PREFIX}/${image}:${DOCKER_TAG}
     for ARCH in ${BUILD_ARCH//,/ }; do
         FORMATTED_ARCH=$(format_archname ${ARCH} tag)
-        ${KUBEVIRT_CRI} manifest add ${DOCKER_PREFIX}/${image}:${DOCKER_TAG} ${DOCKER_PREFIX}/${image}:${DOCKER_TAG}-${FORMATTED_ARCH}
+        TAGGED_IMAGE="${DOCKER_PREFIX}/${image}:${DOCKER_TAG}-${FORMATTED_ARCH}"
+        if skopeo inspect docker://${TAGGED_IMAGE} &>/dev/null; then
+            ${KUBEVIRT_CRI} manifest add ${DOCKER_PREFIX}/${image}:${DOCKER_TAG} ${TAGGED_IMAGE}
+        else
+            echo "Warning: Image ${TAGGED_IMAGE} does not exist, skipping"
```

Comment:
maybe overthinking here, but do we want to have a way to abort the build if an image is missing?